policy/modules/admin/cloudinit.te

```diff
@@ -0,0 +1,108 @@
+policy_module(cloudinit)
+
+########################################
+#
+# Declarations
+#
+
+type cloud_init_t;
+type cloud_init_exec_t;
+init_system_domain(cloud_init_t, cloud_init_exec_t)
+
+type cloud_init_log_t;
+logging_log_file(cloud_init_log_t)
+
+type cloud_init_runtime_t;
+files_runtime_file(cloud_init_runtime_t)
+files_mountpoint(cloud_init_runtime_t)
+
+type cloud_init_state_t;
+files_type(cloud_init_state_t)
+
+########################################
+#
+# Local policy
+#
+
+allow cloud_init_t self:capability { chown dac_override dac_read_search fowner fsetid setgid setuid };
+dontaudit cloud_init_t self:capability { net_admin sys_tty_config };
```

Comment:
How confident are you that the sys_tty_config access is actually unneeded.  It seems to me like the sort of thing a cloud initialization program might want to initialize, but wouldn't be obvious under a lot of testing scenarios, and isn't *that* concerning security wise.

Reply:
Pretty sure, though not 100%.